src/core/index.ts

```diff
@@ -18,11 +18,14 @@ export const batch = batchImport;
 import * as contractsImport from "./contracts";
 export const contracts = contractsImport;
 
+import * as messagesImport from "./messages";
+export const messages = messagesImport;
+
 import * as queueImport from "./queue";
 export const queue = queueImport;
 
-import * as storageImport from "./storage";
-export const storage = storageImport;
+import * as storeImport from "./store";
+export const store = storeImport;
 
 import * as utilitiesImport from "./utilities";
 export const utilities = utilitiesImport;
```

Comment:
is there an 'import * as the filename I imported it + Import and then export it as itself' ? probably not, this is TS/JS!